glean/metrics.yaml

```diff
@@ -332,6 +332,55 @@ sample:
           `authenticationinapp.h`.
         type: string
 
+  auth_account_deletion_step:
```

Comment:
I'd like to mirror `authentication_inapp_step`. My suggestion is `authentication_account_delete_step`.

Reply:
Unfortunately `authentication_account_delete_step` is too long and causes the unit test to throw. The max length seems to be 30 chars.
```
/home/runner/work/mozilla-vpn-client/mozilla-vpn-client/glean/metrics.yaml:
ERROR running glean_parser v6.1.1
    ```
    sample: authentication_account_del_step
    ```
    'authentication_account_del_step' is not valid under any of the given
    schemas
        'authentication_account_del_step' is too long
```